docs/guides/tscircuit-essentials/biscuit-board-laser-ablation.mdx

```diff
@@ -57,12 +57,12 @@ rotating the template in a way that misaligns the coordinates.
   code={`export default () => (
   <board width="20mm" height="20mm" autorouter="laser_prefab">
     <group name="biscuit" pcbX={0} pcbY={0}>
-      <via name="B1" pcbX={-4} pcbY={6} fromLayer="top" toLayer="bottom" netIsAssignable />
-      <via name="B2" pcbX={0} pcbY={6} fromLayer="top" toLayer="bottom" netIsAssignable />
-      <via name="B3" pcbX={4} pcbY={6} fromLayer="top" toLayer="bottom" netIsAssignable />
-      <via name="B4" pcbX={-4} pcbY={0} fromLayer="top" toLayer="bottom" netIsAssignable />
-      <via name="B5" pcbX={0} pcbY={0} fromLayer="top" toLayer="bottom" netIsAssignable />
-      <via name="B6" pcbX={4} pcbY={0} fromLayer="top" toLayer="bottom" netIsAssignable />
+      <via name="B1" pcbX={-4} pcbY={6} fromLayer="top" toLayer="bottom" netIsAssignable outerDiameter={0.8} holeDiameter={0.4} />
+      <via name="B2" pcbX={0} pcbY={6} fromLayer="top" toLayer="bottom" netIsAssignable outerDiameter={0.8} holeDiameter={0.4} />
+      <via name="B3" pcbX={4} pcbY={6} fromLayer="top" toLayer="bottom" netIsAssignable outerDiameter={0.8} holeDiameter={0.4} />
+      <via name="B4" pcbX={-4} pcbY={0} fromLayer="top" toLayer="bottom" netIsAssignable outerDiameter={0.8} holeDiameter={0.4} />
+      <via name="B5" pcbX={0} pcbY={0} fromLayer="top" toLayer="bottom" netIsAssignable outerDiameter={0.8} holeDiameter={0.4} />
+      <via name="B6" pcbX={4} pcbY={0} fromLayer="top" toLayer="bottom" netIsAssignable outerDiameter={0.8} holeDiameter={0.4} />
```

Comment:
as discussed w/ @techmannih , this shouldn't be necessary because there are board defaults